### PR TITLE
Set the group defaults for the allocation

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR sets the group defaults for the default allocation.

### CODE REVIEW

File: skyportal/handlers/api/allocation.py [hunk 1]
```diff
@@ -58,50 +58,61 @@ def get(self, allocation_id=None):
...
+                else:
+                    return self.error(
+                        f"apitype can only be api_classname or api_classname_obsplan, not {apitype}"
+                    )
 
-        allocations = allocations.all()
-        self.verify_and_commit()
-        return self.success(data=allocations)
+            allocations = session.scalars(allocations).all()
+            self.verify_and_commit()
```
Comment (guynir42) [inline skyportal/handlers/api/allocation.py:114]:
```suggestion
            session.commit()
```

Comment (guynir42) [reply skyportal/handlers/api/allocation.py:114]: Are you sure we even need a `commit` here?

**Review by guynir42 — APPROVED** [review]
Some minor comments below:

File: skyportal/handlers/api/allocation.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/allocation.py]: Good point!

**Comment by mcoughlin** [comment]
Thanks @guynir42

**Pull request merged**